app/src/components/market/sections/market_create/steps/items/ask_question_step.tsx

```diff
@@ -322,7 +322,7 @@ const AskQuestionStep = (props: Props) => {
                 disabled={!!loadedQuestionId}
                 name="question"
                 onChange={handleChange}
-                placeholder="What question do you want the world predict?"
+                placeholder="What question do you want the world to predict the answer of?"
```

Comment:
Either that or "What question do you want the world to predict?" @pimato Probably knows the best..

Reply:
I actually prefer: `What would you like to see the world predict?`

Reply:
Thank you @RatanRSur for pointing out the issue :)